app/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set (WEARABLES_XML_FILES xml/FTShoeLeftWearableDevice.xml
                    xml/ICubWearableDevice.xml
                    xml/SkinInsolesWearableDevice.xml
                    xml/XsensSuitWearableDevice.xml
-
+                   xml/applications/WearableDevices-Dumper.xml
```

Comment:
@lrapetti Following #87 can you please add [HumanDynamicsEstimation-WearableDumper.xml](https://github.com/robotology/wearables/blob/master/app/xml/applications/HumanDynamicsEstimation-WearableDumper.xml) for installation.

Reply:
Added in https://github.com/robotology/wearables/pull/92/commits/57648477d8bebbf3c518821623ee555382e61816